src/ui/hooks/useColorModeValue.tsx

```diff
@@ -0,0 +1,14 @@
+import { useTheme } from '@emotion/react';
+import { getColor } from '../../utils/getColor';
+
+const useColorModeValue = (
+  light: string,
+  dark: string,
+  sepia: string
+): string => {
+  const { currentColorMode } = useTheme() as any; // TODO: find out the type for this
```

Comment:
I see, the `useTheme` doesn't know that `currentColorMode` exists on the theme... that's unfortunate but I think this is ok for now in that case.